### DISCUSSION TITLE
Add /Lay & /Crawl

### DISCUSSION THREAD

**Discussion by Strahilchu** [Zrips/CMI#5375] [Feature Requests]
I want to get rid of GSit plugin but I like the /lay and /crawl commands it has, I believe CMI could easily implement those so I decided to request it.

It would be the cherry on top!

**Comment by arturek1666** [comment]
Yeah I would love that as well xD

**Comment by Mrkillero4ka** [comment]
I agree it should be added

**Comment by Recliness** [comment]
Agreed.

**Comment by MelodyYuuka** [comment]
I would love this one too.
